src/top-bar/menu/menu.js

```diff
@@ -52,8 +57,9 @@ Menu.propTypes = {
 export default function Menu({ tabs, activeTab, setActiveTab }) {
   const classes = useStyles(useMuiStyles);
   const { isExtraSmallScreen, isSmallScreen, isLargeScreen } = useScreenSize();
-
```

Comment:
A bit picky, sorry! I like to have a space between the hooks and the rest of the code block. Just keeps things a bit easier to read at a  glance.

Reply:
Truly amazing code review, it makes me think are you a tech lead.
I agree with every comment you have every aspect either improve the syntax of the code or the structure.
I'll be working on my changes so that they align with your ideas.